Assets/Code/VehicleEditor/EditSystem.cs

```diff
@@ -105,8 +128,36 @@ partial class EditSystem : SystemBase {
 
             actionQueue.SetComponent(placementGhost, placeTransform);
 
+            if (hitPart && input.ShowPathToRoot.WasPressedThisFrame())
+            {
+                ShowPathToRoot(ref actionQueue, parentPart: parentPart);
+            }
             if (input.Place.WasPressedThisFrame()) {
-                PlacePart(ref actionQueue, SystemAPI.GetSingletonBuffer<PartsBuffer>()[state.SelectedPart].Value, placeTransform, hitPart ? parentPart : Entity.Null);
+            var partPrefab = SystemAPI.GetSingletonBuffer<PartsBuffer>()[state.SelectedPart].Value;
+                
+            if (hitPart) // If there is no hitpart then symmetry is not needed
+            {
+                    /*
+                        For now Parent Symmetry is the same as root symmetry but only on last "leaf" / or branch idk
+                    and Root is on all branch 
+                        But in the future we might want something more complex that allows us to chose how many branch back do we want to put symmetry on
+                        */
+                    RootSymmetryMode(actionQueue: actionQueue, part: partPrefab, parentPart: parentPart, symmetryCount: sym, placeTransform: placeTransform,MaxIt:it);
+                    /*
+                    var symmetryCount = SystemAPI.GetSingleton<EditSystemData>().SymmetryCount;
```

Comment:
don't leave commented code, just rip it out - if we want it back we can always check the commit history

Reply:
ok